hapi/test/test_credentials.json

```diff
@@ -0,0 +1,3 @@
+{
+  "api_key": "9b1a7bfc-4457-4c9b-b2a2-0cc056352a1c" 
```

Comment:
Where is this api key coming from?

Reply:
According to this like, https://developers.hubspot.com/changelog/upcoming-changes-to-the-public-api-demo-account, we have to create a test account and get API key in the account page. 

I just leave the key file here as an example if people want to run tests again in the future. Maybe I should also include that link in the test_credentials.json.example file.

Reply:
I'd check the permission attached to this key because everybody can see this, other than that, I'm not concerned.

Reply:
Test accounts automatically expire after 90 days. For the permission part, I think it is only for calling APIs being used in this library during tests. Just to be safe, I could remove my key here instead but leave this file for reference later if someone need to run tests.